badge_server/static/css/theme.css

```diff
@@ -200,6 +179,7 @@
     margin-bottom: 0;
 }
 
+/* Clear browser cache: cmd + shift + R  */
```

Comment:
What's this comment about?

Reply:
Oh it's a shortcut for clearing browser cache, which can contain stale css that doesn't reflect recent changes made, but since this isn't actually related to any code, I can remove it.